blockstm/memdb.go

```diff
@@ -0,0 +1,209 @@
+package blockstm
```

Comment:
relatively untested file

Reply:
It has 81.25% coverage according to codecov--though I think it's tested through mvdata_test.go`. I've added some additional tests in `memdb_test.go` as well to push coverage up around object stores.